CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+# Changelog 1.4
+
+## 1.4.0 - 2021-11-03
+
+The NodePortLocal feature is graduated from Alpha to Beta.
+
+### Added
+
+- Support for proxying all Service traffic, including NodePort, LoadBalancer, and ClusterIP traffic. Therefore, running kube-proxy is no longer required. ([#2599](https://github.com/antrea-io/antrea/pull/2599) [#2235](https://github.com/antrea-io/antrea/pull/2235) [#2897](https://github.com/antrea-io/antrea/pull/2897) [#2863](https://github.com/antrea-io/antrea/pull/2863), [@hongliangl] [@lzhecheng])
+  * The feature works for both Linux and Windows
+  * The feature is experimental and therefore disabled by default. Use the `antreaProxy.proxyAll` configuration parameter for the Antrea Agent to enable it
+  * If kube-proxy is removed, the `kubeAPIServerOverride` configuration parameter for the Antrea Agent must be set to access apiserver directly.
+- Add [AntreaIPAM] feature to allow flexible control over Pod IP Addressing. ([#2956](https://github.com/antrea-io/antrea/pull/2956), [@gran-vmv] [@annakhm])
+  * Add new IPPool API to define ranges of IP addresses which can be used as Pod IPs
+  * A Pod's IP will be allocated from the IPPool specified by the `ipam.antrea.io/ippools` annotation of the Pod's Namespace if there is one
+- Add [NodeIPAM] feature to handle the per-Node PodCIDR allocation for clusters where kube-controller-manager does not run NodeIPAMController. ([#1561](https://github.com/antrea-io/antrea/pull/1561), [@ksamoray])
+  * Refer to the [documentation](https://github.com/antrea-io/antrea/blob/main/docs/antrea-ipam.md#running-nodeipam-within-antrea-controller) for instructions on how to configure it.
+- Support for configurable transport interface CIDRs for Pod traffic. ([#2704](https://github.com/antrea-io/antrea/pull/2704), [@Jexf])
+  * Use the `transportInterfaceCIDRs` configuration parameter for the Antrea Agent to choose an interface by network CIDRs
+- Add UDP support for NodePortLocal. ([#2448](https://github.com/antrea-io/antrea/pull/2448), [@chauhanshubham])
+- Add more visibility metrics about connection status for Flow Aggregator. ([#2668](https://github.com/antrea-io/antrea/pull/2668), [@zyiou])
+- Add the `antreaProxy.skipServices` configuration parameter for the Antrea Agent to specify Services which should be ignored by AntreaProxy. ([#2882](https://github.com/antrea-io/antrea/pull/2882), [@luolanzone])
+  * A typical use case is setting `antreaProxy.skipServices` to `["kube-system/kube-dns"]` to make [NodeLocal DNSCache] work when AntreaProxy is enabled.
+- Add support for `ToServices` in the rules of Antrea-native policies to allow matching traffic intended for Services. ([#2755](https://github.com/antrea-io/antrea/pull/2755), [@GraysonWu])
+- Add the `egress.exceptCIDRs` configuration parameter for the Antrea Agent to specify traffic to which SNAT should not be performed. ([#2749](https://github.com/antrea-io/antrea/pull/2749), [@leonstack])
```

Comment:
it should be "traffic for which SNAT should not be performed"

or to be more accurate, maybe:

> Add the `egress.exceptCIDRs` configuration parameter for the Antrea Agent, to specify IP destinations for which SNAT should not be performed on outgoing traffic